met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -64,15 +57,15 @@ export const PreviewBanner = () => {
                             }}
                         >
                             <Button variant="outlined" onClick={() => navigate(`/engagement/form/${engagementId}`)}>
-                                Edit Enagagement
+                                Edit Engagement
                             </Button>
                         </Box>
                         <Button
                             variant="contained"
                             sx={{ marginLeft: '1em' }}
                             onClick={() => handlePublishEngagement()}
```

Comment:
we can maybe consider changing the name here. Since this button either publishes or closes preview. So handlePublishEngagement to do both of those things could be confusing

Reply:
Updated